vault/token_store.go

```diff
@@ -44,6 +44,8 @@ type TokenStore struct {
 	salt *salt.Salt
 
 	expiration *ExpirationManager
+
+	router *Router
```

Comment:
Lets pass in the cubby hole backend directly instead of going through the router